docs/glossary.md

```diff
@@ -0,0 +1,94 @@
+# Glossary
+
+### Attribute
+A characteristic of a material, process, or measurement. Attributes can be a GEMD property, condition, or parameter.
+
+### Attribute Template
+Attribute templates define a domain concept (e.g. density, yield strength), and include bounds to define acceptable values for a given attribute.
+
+### Bounds
+Bounds are used in object and attribute templates. Object templates define the widest bounds, and attribute templates bounds further constrain values to a tighter range or set.  Bounds may be real or categorical.
+
+### Categorical Bounds
+Bounds that are defined by a list of categories.
+
+### Condition Attribute
+Environmental variables (typically measured) that may affect a process or measurement, such as temperature or pressure.
+
+### Data Object
+GEMD materials, processes, measurements, and ingredients. Data objects along with attributes are linked to form a complete description of a final material and how it was created.
+
+### GEMD
+Graphical Expression of Materials Data, an open source format for storing materials data.
+
+### Ingredient Object
+A  data object that annotates a material with information related to its usage in a process. Ingredient objects are used define how much of a material is used in a new process. Note that 'ingredient' and 'material' object definitions are not interchangeable.
+
+### Material Object
+GEMD data object that describes a physical substance that is generated by a process and is characterized by its properties. GEMD does not allow material objects to exist in isolation, since fully describing a material data point requires, at minimum, identifying how the material was procured.  
+
+### Material Run
+An actual instance of a material.
+
+### Material Spec
+Material spec objects are used to integrate information from external references, typically a data sheet. Reference information
+is optionally encoded via PropertyAndCondition attributes. 
+
+### Material Template
+Define how to validate any PropertyAndConditions attributes, if they exist on a linked material spec. 
+
+### Measurement Object
+A data object used to describe an operation that measures or characterizes properties of a material.
+
+### Measurement Run
+A recorded measurement for a specific material run object.
+
+### Measurement Spec
+Defines the intent for how a measurement should be performed, including any parameters and conditions. 
+
+### Measurement Template
+Defines the widest set of allowable bounds for associated conditions, parameters and properties. 
+
+### Nominal Real Value
+A value type in which the value is recorded as a specific number, with no information about the uncertainty of that number.
```

Comment:
```suggestion
A value type in which the value is recorded as a specific number with a unit, with no information about the uncertainty of that number.
```

Do we want to include all Value types?  There's no discussion of composition, categorical, integer, or molecular structure value types.